scripts/test.sh

```diff
@@ -169,14 +169,16 @@ case "$TARGET" in
   pushd tests/react-test-app
 
   npm install
-  open_chrome
-  start_packager
```

Comment:
I think we still want to run the packager internally? Otherwise I think it pops open on a new terminal on ci and we lose control over it.